TimerGUI.pyw

```diff
@@ -1,120 +1,97 @@
-#GUI and commands code
-
-import tkinter as tk
-import time
-import math
+#program config (in seconds):
+#time before first window pop's out 
+programInitialDelay = 480
+#time before first window pop's out again after clicking NO
+shortHideButtonDelay = 120
+# !!!CHANGE NOT ADVISED!!! time before "change log file" info pop's out again
+timerDelay = 3600
 
-#first window creator 
-def createInitialWindow():
-    #first window constructor
-    initialWindow = tk.Tk()
-    initialWindow.title('Timer lasera')
+#GUI and commands code
+import tkinter as tk, time, math
 
-    #Set window to appear in the middle
+def createWindow():
+    
+    window = tk.Tk()
+    window.title('Timer lasera')
+    
     windowWidth = 330
     windowHeight = 122
-    screenWidth = initialWindow.winfo_screenwidth()
-    screenHeight = initialWindow.winfo_screenheight()
+    screenWidth = window.winfo_screenwidth()
+    screenHeight = window.winfo_screenheight()
 
     x_pos = math.floor((screenWidth - windowWidth) / 2)
-    y_pos = math.floor(((screenHeight - windowHeight) / 2) * 0.80)
+    y_pos = math.floor(((screenHeight - windowHeight) / 2) * 0.80) #multiply by 0.8 to make the window appear more to the top of the screen
+
+    window.geometry(f"{windowWidth}x{windowHeight}+{x_pos}+{y_pos}")
+    return window
 
-    initialWindow.geometry(f"{windowWidth}x{windowHeight}+{x_pos}+{y_pos}")
+def createLabel(window, text):
+    label = tk.Label(
+        window,
+        text = text,
+        width = 22,
+        height = 2,
+        font = 'Times 18 bold',
+        padx = 10,
+        pady = 10,
+        justify ='center')
+    return label
+
+def createButton(window, buttonText, bgColor, buttonCommand):
+    button = tk.Button(
+        window,
+        text = buttonText,
+        bg = bgColor,
+        justify = 'center',
+        font = 'Times 18 bold',
+        command = buttonCommand)
+    return button
+
+#first window creator 
+def createInitialWindow():
+    initialWindow = createWindow()
 
     #NO button command
     def shortWindowHide():
         initialWindow.withdraw()
-        time.sleep(2)  #2 minuty = 120
+        time.sleep(shortHideButtonDelay)
         initialWindow.deiconify()
 
     #YES button command
     def timerStart():
         initialWindow.withdraw()
-        time.sleep(2) #1h = 3600
+        time.sleep(timerDelay)
         createRecurringWindow()
         initialWindow.destroy()
 
     #Labels and Buttons for first window
-    isLoggerOn = tk.Label(
-        initialWindow,
-        text = 'Czy włączono logger lasera?',
-        width = 22,
-        height = 2,
-        font='Times 18 bold',
-        padx=10,
-        pady=10,
-        justify='center'
-        )
+    isLoggerOn = createLabel(initialWindow, 'Czy włączono logger lasera?')
```

Comment:
In python strings are generally used with double quotes for text, and single quotes for identifiers/paths. No functional difference, just good practice.